### DISCUSSION TITLE
Asking for installation guide & how to use components

### DISCUSSION THREAD

**Discussion by hey6** [kujangstack/uniui#7] [Q&A]
How do I use the components from this library? Why is there no example or installation guide in the readme?

**Comment by hiriski** [comment]
Currently, there is no guide on how to install and also an example of how to use the component because this library is under development and may change a lot in the future.

If you want to try out the dev version components you can use this configuration theme in your react native project 

Create `unistyles.ts`
```ts
import { UnistylesRegistry } from 'react-native-unistyles'
import { theme_lightTheme, theme_darkTheme, theme_breakpoints, ThemeBreakpoints, Themes } from '@kujang/uniui'

declare module 'react-native-unistyles' {
  export interface UnistylesBreakpoints extends ThemeBreakpoints {}
  export interface UnistylesThemes extends Themes {}
}

UnistylesRegistry.addBreakpoints(theme_breakpoints)
  .addThemes({
    light: theme_lightTheme,
    dark: theme_darkTheme,
  })
  .addConfig({
    adaptiveThemes: true,
    initialTheme: 'light',
  })

````

the import it from your root code eg. `App.tsx`
```ts
import './unistyles'
```
